project_generator/exporters/uvision_definitions.py

```diff
@@ -119,12 +119,14 @@ def get_mcu_definition(self, name):
             }
         },
     }
+    # alias mbed standard names to their CPUs:
+    mcu_def['K64F'] = mcu_def['MK64FN1M0xxx12']
```

Comment:
I totally agree about more sane naming, this was just a hack so that for the target I cared about the name was the mbed standard one.